src/frontend-pwa/src/App.tsx

```diff
@@ -3,13 +3,15 @@ import { useState, useEffect } from 'react';
 import axios from 'axios';
 import Header from './components/Header/Header';
 import Footer from './components/Footer/Footer';
-import ViewRouter from './routes/ViewRouter';
 import SplashScreen from './components/SplashScreen/SplashScreen';
+import Eula from './views/Eula/Eula';
+import ViewRouter from './routes/ViewRouter';
 import LocationsArray from './Type/LocationsArray';
 import constants from './constants/Constants';
 
 function App() {
   const [isLoading, setIsLoading] = useState(true);
+  const [eula, setEula] = useState(false);
```

Comment:
Maybe something a little more descriptive like `eulaAccepted and setEulaAccepted`?